website/docs/r/service_networking_application_load_balancer_frontend.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Service Networking"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_application_load_balancer_frontend"
+description: |-
+  Manages an Application Gateway for Containers Frontend.
+---
+
+# azurerm_application_load_balancer_frontend
+
+Manages an Application Gateway for Containers Frontend.
+
+## Example Usage
+
+```hcl
+resource "azurerm_application_load_balancer" "example" {
+  name                = "example"
+  resource_group_name = "example"
+  location            = "West Europe"
+}
+
+resource "azurerm_application_load_balancer_frontend" "example" {
+  name                         = "example"
+  location                     = "West Europe"
+  application_load_balancer_id = azurerm_application_load_balancer.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Application Gateway for Containers Frontend. Changing this forces a new resource to be created.
+
+* `location` - (Required) The Azure Region where the Application Gateway for Containers Frontend should exist. Changing this forces a new resource to be created. Available Regions could be found in [here](https://learn.microsoft.com/en-us/azure/application-gateway/for-containers/overview#supported-regions).
```

Comment:
The word region can be lowercase.  In the last sentence, I'd change could to can and remove the word in.

Available regions can be found [here](link).